Marsiple/App/Posts/PostTableViewCell.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  PostTableViewCell.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 16/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class PostTableViewCell: UITableViewCell {
+    private let title = UILabel.autolayoutView()
+    private let body = UILabel.autolayoutView()
```

Comment:
I would definitely provide more context and descriptiveness within the variable naming. `body` alone does not really explictily detail what this variable does.

`bodyLabel` would be a lot better, as it fully describes the variable's purpose.